basis/listener/listener.factor

```diff
@@ -24,8 +24,13 @@ H{
 M: object prompt.
     nip prompt-style get-global format bl flush ;
 
+: with-ctrl-break ( quot -- )
+    enable-ctrl-break
+    [ disable-ctrl-break ] [ ] cleanup ; inline
+
 : parse-lines-interactive ( lines -- quot/f )
-    [ parse-lines ] with-compilation-unit ;
+    [ [ parse-lines ] with-ctrl-break
```

Comment:
Hah, I've been promising 0.98 for awhile (even longer than that!), but real life has gotten somewhat in the way.  Plus, the two main bugs I want to get fixed are unfixed (Slow GTK resize on @erg's VM, and OpenGL initialization on macOS).